projects/devices/robotiq/protos/Robotiq3fGripper.proto

```diff
@@ -76,16 +76,17 @@ PROTO Robotiq3fGripper [
               device [
                 RotationalMotor {
                   name "finger_1_joint_1"
-                  maxVelocity 100
-                  maxPosition 3.1416
-                  maxTorque 100
+                  maxVelocity 100.0
+                  maxPosition 1.1723
```

Comment:
Can you explain the difference with https://github.com/ros-industrial/robotiq/blob/kinetic-devel/robotiq_3f_gripper_visualization/cfg/robotiq-3f-gripper_articulated.urdf#L109 ?

Reply:
Ok I see, this comes from https://github.com/ros-industrial/robotiq/blob/66961ec6b6c9c493f13410d16ce32cea1648babf/robotiq_3f_gripper_visualization/cfg/robotiq-3f-gripper_articulated_macro.xacro#L49 right?

Reply:
The finger limits are specifically from here: https://github.com/ros-industrial/robotiq/blob/66961ec6b6c9c493f13410d16ce32cea1648babf/robotiq_3f_gripper_visualization/cfg/robotiq-3f-gripper_finger_articulated_macro.xacro
Since I changed it so either max or min position is 0, one value is subtracted from the other. Here  lower="0.0495" upper="1.2218" turns to  lower="0.0" upper="1.1723"